srcalc/src/main/java/gov/va/med/srcalc/vista/VistaPatientDao.java

```diff
@@ -30,6 +30,23 @@ public String getDescription()
             return fDescription;
         }
     }
+    
+    /**
+     * This is a list of the only health factors that should be displayed for the users.
+     * This list was created by the NSO.
```

Comment:
This is pretty minor, but since this is open source, I would remove the reference to the NSO. Maybe just say "the VA".